src/tools/yarprobotstatepublisher/include/robotstatepublisher.h

```diff
@@ -47,6 +47,8 @@ class YARPRobotStatePublisherModule : public yarp::os::RFModule
     yarp::dev::PolyDriver       m_ddtransformclient;
     yarp::dev::IFrameTransform       *m_iframetrans;
 
+    std::string m_tfPrefix;
```

Comment:
It would be really helpful if you could add a clear documentation about this parameter.